cfgov/jinja2/v1/owning-a-home/loan-options/FHA-loans/index.html

```diff
@@ -0,0 +1,144 @@
+        </section>
+
+        <div class="col-3">
+          <h4 class="small-heading">Mortgage insurance</h4>
+          <p class="sans">Required for <strong>all</strong> FHA loans. <br><a href="#mortgage-insurance">More on mortgage insurance</a>.</p>
+        </div>
+      </div>
+
+      <section id="mortgage-insurance" class="l-bump sans">
+          <h3>Mortgage insurance: what you need to know</h3>
+
+           <p><strong>Mortgage insurance helps you get a loan you wouldn’t otherwise be able to.</strong> If you can’t afford a 20 percent down payment, you will likely have to pay for mortgage insurance. You may choose to get a conventional loan with private mortgage insurance (PMI), or an FHA, VA, or USDA loan.</p>
+          <p><strong>Mortgage insurance usually adds to your costs.</strong> Depending on the loan type, you will pay monthly mortgage insurance premiums, an upfront mortgage insurance fee, or both. </p>
+          <p><strong>Mortgage insurance protects the lender if you fall behind on your payments. It does not protect you.</strong> Your credit score will suffer and you may face foreclosure if you don’t pay your mortgage on time.</p>
+          <p><a href="/askcfpb/1953/what-is-mortgage-insurance-and-how-does-it-work.html" target="_blank">Learn more about mortgage insurance</a>.</p>
+      </section>
+      {% include "_layouts/ratings-form.html" %}
+    </section> <!-- /.content_main -->
+
+    <aside class="content_sidebar sans">
+      <div class="block
+                  block__flush-top">
+        {{ social_media.render( {
+              "twitter_text": "Is an FHA mortgage loan right for you? The @CFPB helps you decide.",
+              "email_title": "Is an FHA mortgage loan right for you?",
+              "email_text": "The CFPB helps you understand the pros and cons of FHA loans so you can decide what’s right for you.",
+              "email_signature": "-- From the CFPB",
+              "linkedin_title": "Is an FHA mortgage loan right for you?",
+              "linkedin_text": "@CFPB helps you understand all the details and decide what’s right for you. Via @CFPB"
+        } ) }}
+      </div>
+
+      {% include "_layouts/oah-email-signup.html" %}
+
+      <div class="block">
+        <header class="m-slug-header">
+            <h2 class="a-heading">
+                Other loan types
+            </h2>
+        </header>
+        <ul class="m-list m-list__spaced m-list__unstyled">
+          <li class="m-list_item">
+            <a class="a-link
+                      a-link__jump
+                      cf-icon
+                      cf-icon__after
+                      cf-icon-right"
+               href="../conventional-loans/">
+              <span class="jump-link_text">Conventional loans</span>
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/3165/commits/adea54f8ecb4f983b08882aaf842c2e812e402bf.